### PR TITLE
Fix schema serialization and sitemap settings regressions

### LINKED ISSUE #330: [Bug] Custom Post Type tabs broken in Sitemap Settings since v1.0.266
**Describe the bug**
Since v1.0.266 (commit `4e26297`), custom post type (CPT) tabs in **Rank Math SEO → Sitemap Settings** display with **blank labels** in the sidebar navigation. Additionally, CPT sitemaps that were working on v1.0.265 return **404 errors** after upgrading, until the "Include in Sitemap" toggle is manually disabled and re-enabled.

This affects all sites with custom post types across multiple installations. Downgrading to v1.0.265 (commit `0b67698`) resolves both issues completely.

**To Reproduce**

1. Have one or more custom post types registered (e.g., `services`, `team`, `testimonials`) with `public => true` and sitemaps working on v1.0.265
2. Update Rank Math SEO to v1.0.266 or later
3. Go to **Rank Math SEO → Sitemap Settings**
4. Observe the left sidebar: custom post type tabs show icons but **no labels**
5. Visit the CPT sitemap URL directly (e.g., `example.com/services-sitemap.xml`) — it returns **404**
6. Navigate to `?page=rank-math-options-sitemap&view=sitemap-post-type-services` — the page content area is blank (URL mismatch)

**Workaround:** Click the blank CPT tab icon (the fields inside load correctly), toggle "Include in Sitemap" **off**, save, then toggle it back **on** and save again. The 404 resolves, but the sidebar label remains blank.

**Expected behavior**

- All custom post types should display their labels in the Sitemap Settings sidebar
- Direct URLs to CPT sitemap settings tabs should load the correct tab content
- Existing CPT sitemap settings should carry over from v1.0.265 without requiring manual re-save
- XML sitemaps for custom post types should continue working after the update without intervention

**Screenshots**
*(attach screenshot showing blank CPT labels in the Sitemap Settings sidebar — tabs with icons but no text)*

**Desktop (please complete the following information):**

- OS: Windows 11 / macOS
- Browser: Chrome 147.0.7727.56 
- Version: Rank Math SEO v1.0.266+ (broken) / v1.0.265 (working)

### PR DESCRIPTION
**Pull request by xndvaz**
## Summary
- Preserve booleans and numeric values when sanitizing JSON-LD schema output.
- Avoid setup wizard warnings when a post type does not provide the article type schema field.
- Align the compiled sitemap settings bundle with the existing source tab name format.

## Validation
- php -l includes/modules/schema/class-jsonld.php
- php -l includes/admin/wizard/class-schema-markup.php
- php -l includes/modules/analytics/class-ajax.php
- php -l includes/modules/analytics/class-analytics.php
- php -l includes/modules/analytics/class-db.php
- php -l includes/modules/analytics/workflows/class-oauth.php
- JSON-LD smoke test preserving booleans and numbers
- Wizard smoke test with missing product article type field
- Confirmed no stale name:"post-type-"+e remains in the sitemap settings bundle
- git diff --check

Fixes #318
Fixes #319
Fixes #330

### CODE REVIEW

**Comment by xndvaz** [comment]
Closing this PR as I am no longer maintaining contributions to this project. Thanks.
